Sources/locgen-swift/URLEphemeralSession.swift

```diff
@@ -0,0 +1,12 @@
+//
+//  URLEphemeralSession.swift
```

Comment:
To be consistent I'd name this file as `URLSession+Extensions` as proposed in [this](https://github.com/Holdapp/locgen-swift/blob/main/Sources/locgen-swift/Foundation%2BExtensions.swift) file but besided this looks good to me

Reply:
changed here fbff8316848b80cedaf3596c868457aff2005a72

Reply:
I need an approve to merge this PR, would you mind? 😅

Reply:
Oh, it's already approved 😬 sorry